src/dotnet-user-secrets/project.json

```diff
@@ -4,7 +4,7 @@
     "warningsAsErrors": true,
     "keyFile": "../../tools/Key.snk"
   },
-  "description": "DNX tool to manage user secrets.",
+  "description": "dotnet tool to manage user secrets.",
   "repository": {
```

Comment:
while you're at this, can you please remove the repository property? It's not used anymore